academy/4-my-own-chain/create-handling-exer.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Exercise - Message Handler - Create and Save a Game Properly
+order:
```

Comment:
@coldice How do we handle the ordering here?

Reply:
It is ignored 😈